tests/application/test_routes.py

```diff
@@ -0,0 +1,316 @@
+                "cms.create_table",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/create-table"
+        )
+        assert (
+            get_url(
+                "cms.delete_chart",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/delete-chart"
+        )
+        assert (
+            get_url(
+                "cms.delete_table",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/delete-table"
+        )
+        assert (
+            get_url(
+                "cms.save_chart_to_page",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/save-chart"
+        )
+        assert (
+            get_url(
+                "cms.save_table_to_page",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/save-table"
+        )
+        assert (
+            get_url(
+                "cms.delete_dimension",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/delete"
+        )
+        assert (
+            get_url(
+                "cms.edit_dimension",
+                topic_slug="topic",
+                subtopic_slug="subtopic",
+                measure_slug="measure",
+                version="1.0",
+                dimension_guid="dimension-guid",
+            )
+            == "/cms/topic/subtopic/measure/1.0/dimension-guid/edit"
+        )
+
+        # Make sure that the routes we've tested above are all of the routes that our app has registered with Flask
+        # i.e. that we haven't added a new view and not updated this test to lock the URL path in place.
+        assert routes == set(rule.endpoint for rule in app.url_map.iter_rules() if rule.endpoint.split(".")[0] == "cms")
```

Comment:
Maybe add a message here as an argument to the assertion, as well as the comment?

Reply:
👍